README.md

```diff
@@ -20,12 +24,23 @@ Two experiments are carried out:
         + Bruteforce messaging with RTC (run time compilation)
         + Spatial3D messaging 
         + Spatial3D messaging with RTC (run time compilation) 
+
+[![Fixed Density Benchmark](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/fixed-density--volume--step-s--model--all.png)](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/fixed-density--volume--step-s--model--all.png)
+[![Fixed Density Benchmark Zoomed](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/fixed-density--volume--step-s--model--zoomed.png)](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/fixed-density--volume--step-s--model--zoomed.png)
```

Comment:
The Y axis is incorrecr on the committed zoomed figure due to the change to seconds (it's now zoomed out not zoomed in).

The appropriate value will need adjusting in the plotting script (0.2 would be the intended value, though this might be a little trial and error as the data generated is newer than used). 
It is to show the RTC series much more clearly than on the combined graph.

Reply:
Value of 0.1 used and tested on v100 results. It could even be smaller.